DiverseSelector/test/test_metrics.py

```diff
@@ -0,0 +1,114 @@
+# -*- coding: utf-8 -*-
+# The DiverseSelector library provides a set of tools to select molecule
+# subset with maximum molecular diversity.
+#
+# Copyright (C) 2022 The QC-Devs Community
+#
+# This file is part of DiverseSelector.
+#
+# DiverseSelector is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 3
+# of the License, or (at your option) any later version.
+#
+# DiverseSelector is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, see <http://www.gnu.org/licenses/>
+#
+# --
+
+"""Testing for the distance, similarity, and diversity algorithms."""
+
+# todo: fix this later
+# noqa: F401
+from DiverseSelector.metric import  (
+                                    bit_tanimoto,
+                                    distance_to_similarity,
+                                    pairwise_similarity_bit,
+                                    tanimoto,
+                                    ComputeDistanceMatrix,
+                                    )
+
+from DiverseSelector.test.common import (
+                                    bit_dice,
+                                    euc_bit,
+                                    bit_cosine,
+                                    cosine,
+                                    dice,
+                                    )
+
+import numpy as np
+from numpy.testing import assert_equal
+
+
+# each row is a feature and each column is a molecule
+sample1 = np.array([[4, 2, 6],
+                    [4, 9, 6],
+                    [2, 5, 0],
+                    [2, 0, 9],
+                    [5, 3, 0]])
+
+
+# each row is a molecule and each colume is a feature (scipy)
+sample2 = np.array([[1, 1, 0, 0, 0],
+                    [0, 1, 1, 0, 0],
+                    [0, 0, 0, 1, 0],
+                    [0, 0, 0, 0, 1]])
+
+sample3 = np.array([[1, 4],
+                    [3, 2]])
+
+sample4 = np.array([[1, 0, 1],
+                    [0, 1, 1]])
+
+
+def test_Compute_Distance_Matrix_Euc_bit():
```

Comment:
We normally use lower case for functions names and keep upper case for python Class.